### ISSUE TITLE
Incorrect Output

### ISSUE THREAD

**Issue by skipjack** [developit/microbundle#583]
Not sure if this a bug with this package or some underlying dependency like `rollup`. The following module...

``` js
export * from 'd3-drag'
export * from 'd3-zoom'
export * from 'd3-force'
export * from 'd3-brush'
export * from 'd3-selection'
```

gets compiled to the following output in v0.12.0-next.8...

``` js
import{forceCollide as u,forceManyBody as a,forceX as l,forceY as f,forceSimulation as h,forceLink as d,select as v,event as p,zoom as y,zoomIdentity as m,drag as b,brush as g}from"d3-drag";
import"d3-zoom";
import"d3-force";
import"d3-brush";
import"d3-selection";
```

I think the issue is related to tree shaking. The partial `import`s like `forceCollide` aren't even a part of `d3-drag`. Looks like it's just treating the first `export * from ...` as the main module. I think I can work around this by using partial imports within each module but the syntax I'm using is valid.

**Comment by developit** [comment]
Interesting. Are you inlining those modules?

**Comment by skipjack** [comment]
No they're correctly being treated as externals, the variable binding is just butchered.

**Comment by skipjack** [comment]
Yeah so moving to more explicit partial imports from the consumers did fix the issue. There must be something with `export * from ...` that's not handled correctly. @developit I guess this is kind of similar to #546 in that I'm using the latest export features however in this case it is already [out of the proposal phase][1] as far as I can tell.

[1]: https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export

**Comment by skipjack** [comment]
For more context, I essentially had the following...

__src/d3.js__

``` js
export * from 'd3-drag'
export * from 'd3-zoom'
export * from 'd3-force'
export * from 'd3-brush'
export * from 'd3-selection'
```

__src/other.js__

``` js
import * from './d3.js'
```

**Comment by developit** [comment]
This should be fixed as of 0.13 due to #738.
